privacyidea/lib/token.py

```diff
@@ -1685,6 +1685,30 @@ def set_description(serial, description, user=None):
     return len(tokenobject_list)
 
 
+@log_with(log)
+@check_user_or_serial
+def set_email(serial, email, user=None):
```

Comment:
This is a tokentype specific field. Such functionality should not be added here.